query_execution/Worker.hpp

```diff
@@ -103,6 +104,18 @@ class Worker : public Thread {
   void run() override;
 
  private:
+  /**
+   * @brief Send the response WorkOrder completion message.
+   *
+   * @param receiver The id of the TMB client which should receive the response.
+   * @param op_index The index of the operator to which the WorkOrder belongs.
+   * @param isRebuildWorkOrder True if it is a RebuildWorkOrder. Otherwise
+   *        false.
+   **/
+  void sendWorkOrderCompleteMessage(const tmb::client_id receiver,
+                                    const std::size_t op_index,
+                                    const bool isRebuildWorkOrder);
```

Comment:
The style of variable naming is different than our usual standard which is - ``is_rebuild_workorder``